src/utils/helpers.js

```diff
@@ -0,0 +1,13 @@
+
+export const chunkArray =(array, size) => {
```

Comment:
I don't see a use for this anymore. At least, I'm not using it for map generation. :)